src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -838,6 +865,8 @@ mobile_apis::Result::eType CommandRequestImpl::PrepareResultCodeForResponse(
   mobile_apis::Result::eType result_code = mobile_apis::Result::INVALID_ENUM;
   if (IsResultCodeUnsupported(first, second)) {
```

Comment:
Should call `IsResultCodeUnsupported(first, second) || IsResultCodeUnsupported(second, first)`

Reply:
@jacobkeeler checks reworked to avoid double checks. And visa versa checks was added.